[BUG] Cannot find module '/app/scripts/migrate-legacy.mjs' when migrating the database

**Describe the bug**
The database migration script mentioned in the setup page does not exists.

**To Reproduce**
Steps to reproduce the behavior:
1. Pull a new version and re-build the container
2. Go to http://localhost:3000/setup
3. See the "💡 Upgrading from an older version?" message with the following migration command: `docker compose exec wwv node scripts/migrate-legacy.mjs`
4. Run the command in the repository
5. See error

Resetting the database with `docker compose down -v` does not make the upgrade message go away.

**Expected behavior**
The upgrade command should work

**Screenshots/Logs**
```
-> % docker compose exec wwv node scripts/migrate-legacy.mjs
node:internal/modules/cjs/loader:1505
  throw err;
  ^

Error: Cannot find module '/app/scripts/migrate-legacy.mjs'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1502:15)
    at wrapResolveFilename (node:internal/modules/cjs/loader:1073:27)
    at defaultResolveImplForCJSLoading (node:internal/modules/cjs/loader:1097:10)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1118:12)
    at Module._load (node:internal/modules/cjs/loader:1287:25)
    at wrapModuleLoad (node:internal/modules/cjs/loader:255:19)
    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:154:5)
    at node:internal/main/run_main_module:33:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v26.1.0
```

**Environment (please complete the following information):**
 - OS: Gentoo Linux
 - Browser: Firefox
 - WorldWideView Edition: local docker compose

--- a/src/app/setup/page.tsx
+++ b/src/app/setup/page.tsx
@@ -33,13 +33,6 @@ export default function SetupPage() {
                 <h1 className={styles.title}>Welcome to WorldWideView</h1>
                 <p className={styles.subtitle}>Create your admin account to get started</p>
 
-                {/* Legacy Data Detection Banner */}
-                <div className={styles.legacyNotice}>
-                    <strong>💡 Upgrading from an older version?</strong>
-                    <p>We've detected legacy SQLite data. Once your new Postgres database is connected, you can import your favorites by running the migration command in your terminal.</p>
-                    <code>docker compose exec wwv node scripts/migrate-legacy.mjs</code>
-                </div>
-
                 <form onSubmit={handleSubmit} className={styles.form}>
                     <label className={styles.label} htmlFor="name">
                         Display Name

--- a/src/app/setup/setup.module.css
+++ b/src/app/setup/setup.module.css
@@ -103,27 +103,3 @@
 .button:hover { opacity: 0.9; }
 .button:disabled { opacity: 0.5; cursor: not-allowed; }
 
-.legacyNotice {
-    margin: 1.5rem 0;
-    padding: 1rem;
-    background: rgba(255, 191, 0, 0.05);
-    border: 1px solid rgba(255, 191, 0, 0.3);
-    text-align: left;
-    font-size: 0.8rem;
-    color: #ffbf00;
-}
-
-.legacyNotice strong {
-    display: block;
-    margin-bottom: 0.4rem;
-}
-
-.legacyNotice code {
-    display: block;
-    margin-top: 0.75rem;
-    padding: 0.5rem;
-    background: #000;
-    color: #0f0;
-    font-family: 'JetBrains Mono', monospace;
-    word-break: break-all;
-}

